dcos/marathon.py

```diff
@@ -22,10 +22,9 @@ def create_client(config=None):
         config = util.get_config()
 
     marathon_url = _get_marathon_url(config)
-    timeout = config.get('core.timeout', http.DEFAULT_TIMEOUT)
 
     logger.info('Creating marathon client with: %r', marathon_url)
-    return Client(marathon_url, timeout)
+    return Client(marathon_url)
```

Comment:
I would like that the functionality in the `dcos` package works outside of the context of the CLI. I would like developers to be able to use `dcos.marathon` in their script without needed the CLI. I know that it doesn't work now but I want to make sure we don't add more CLI dependencies to it.

For example, instantiating and using dcos.marathon.Client requires `dcos.http` which requires reading the environment variable DCOS_CONFIG and the file it points to.